bitnami/grafana/Chart.yaml

```diff
@@ -33,4 +31,4 @@ maintainers:
 name: grafana
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana
-version: 9.3.2
+version: 9.3.0
```

Comment:
There is something wrong with the version. Not sure if we should bump the major or the minor...

Reply:
There's no backwards incompatibility, that's why I bumped the minor version. That said, it's true that it's needed an extra step to have the same plugin configuration after upgrading.